src/components/People/index.tsx

```diff
@@ -155,7 +177,8 @@ export default function PeopleList(): JSX.Element {
 
   const removeSelectedPeopleFromOrg = async () => {
     if (selectedOrganization) {
-      if (selectedPeopleRows.length === selectedOrganization.totalUsers) {
+      const removableUsers = selectedPeopleRows.filter((row) => !isTfContact(row.role));
+      if (removableUsers.length === totalUsers) {
```

Comment:
Does totalUsers include TfContacts? This will never be the same is removableUsers remove the TfContacts?

Reply:
does not include tfContacts (will comment)